api/proto/metrics/metrics.proto

```diff
@@ -17,11 +17,46 @@ enum ZmetricTypes {
   ZmApp = 3;
 }
 
+// memoryMetric - DEPRECATED. DO NOT USE THIS.
 message memoryMetric {
   uint32 usedMem = 2;            //in MBytes
   uint32 availMem = 3;           //in MBytes
-  double usedPercentage = 4;
-  double availPercentage = 5;
+  double usedPercentage = 4; // Obsolete - No Longer Used.
+  double availPercentage = 5; // Obsolete - No Longer Used.
+}
+
+// App Memory Metric
+message AppMemoryMetric {
+    // allocatedMB - Memory allocated to app instance by EVE. In Steady State,
+    //  this is expected to be same as in App instance config. Eve should
+    //  report what it actually allocated / reserved instead of what is configured.
```

Comment:
Also, for this field you need to state what is reported when the application is being deployed but has not yet booted. In that case the config API specifies some memory which is allocated to the application, but the application doesn't yet have it.
I think the clearest way to do that is to take the number from the config API (and add any slack we might tack on for qemu). Is that what you had in mind?

Reply:
If the intent with the sentence is to report what we might tack on for qemu it would make sense to explicitly say so e.g.,
EVE might allocate additional memory for the virtual device model (e.g., qemu) in which case that is added to the allocatedMB. Thus in that case allocatedMB will be larger than the memory specified in the config API.

Reply:
Made changes as discussed